index.bs

```diff
@@ -271,16 +271,16 @@ Report image Element Timing {#sec-report-image-element}
     1. Let |exposedElement| be the result of running [=get an element=] with |element| and |document| as input.
     1. If |exposedElement| is not null, call the <a>potentially add a LargestContentfulPaint entry</a> algorithm with |intersectionRect|, |imageRequest|, |renderTime|, |loadTime|, |element|, and |document|.
     1. If |element|'s "<code>elementtiming</code>" content attribute is absent or if its value is the empty string, then abort these steps.
-    1. Create a {{PerformanceElementTiming}} object |entry|.
-    1. Set |entry|'s <a>request</a> to |imageRequest|.
-    1. Set |entry|'s <a>element</a> to |element|.
-    1. Set |entry|'s {{PerformanceEntry/name}} to the {{DOMString}} "image-paint".
-    1. Set |entry|'s {{renderTime}} to |renderTime|.
-    1. Set |entry|'s {{loadTime}} to |loadTime|.
-    1. Set |entry|'s {{intersectionRect}} to |intersectionRect|.
-    1. Set |entry|'s {{identifier}} to |element|'s "<code>elementtiming</code>" content attribute.
-    1. Set |entry|'s {{PerformanceElementTiming/naturalWidth}} and {{PerformanceElementTiming/naturalHeight}} by running the same steps for an <{img}>'s {{img/naturalWidth}} and {{img/naturalHeight}} attribute getters, but using |imageRequest| as the image.
-    1. Set |entry|'s {{id}} to |element|'s "<code>id</code>" content attribute.
+    1. Create and initialize a {{PerformanceElementTiming}} object |entry|.
+        1. Initialize |entry|'s <a>request</a> to |imageRequest|.
```

Comment:
I believe 'set' is more commonly used in specs even when it is the first time the value is being set. Is there any guidance you found indicating that 'initialize' is the correct word to use? Otherwise I'd keep 'initialize' in the attribute setting ("Create and initialize" seems fine to me).

Reply:
Thanks for the feedback. I definitely agree with you -- set is the better word here. However, what concerns me is that earlier in the spec we refer to the value at initialization as the value that the getter must return. If we clarify that these are the values at initialization, then I am perfectly happy! Thanks again for the feedback!

Reply:
Ok, don't have strong opinion on set vs initialize here. Merging this, thanks for the PR!